example-overrides/local-overrides-centos8-tripleo-train.yaml

```diff
@@ -0,0 +1,10 @@
+standalone_host: <standalone FQDN>
+# Both public_api and control_plane_ip have to be the same in OSP16 for Pacemaker
+public_api: <IP address used to reach the node>
+control_plane_ip: <Same IP address used to reach the node>
+tripleo_repos_branch: train
+cip_config:
```

Comment:
Is this used automatically by something?

Reply:
this is something I'll refactor later to use the tripleo operator for container image prepare role from there. For now, this is just an example on how to override the default we have in dev-install, and yes this is used as an environment input for when deploying standalone.

Reply:
Ok, so it wouldn't be used by anything in this commit?

Reply:
this is an overrides example if you plan to deploy on centos/RDO, train version (upstream for OSP16), so yes this should be used.

Reply:
It's used by `tripleo.operator.tripleo_repos`?

Reply:
no, it creates an environment file that is used by standalone.